rpgJavaInterpreter-core/src/test/resources/ERROR23.rpgle

```diff
@@ -0,0 +1,16 @@
+     V* ==============================================================
+     D* 09/05/24
+     D* Purpose: Must fire the following errors
+     D* line 9 - Factor 1 can't be null
+     D* line 14 - Factor 2 can't be null
+     V* ==============================================================
+
+     C                   SETOFF                                           55
+     C     *IN55         DOUEQ
```

Comment:
Ask rpgle developer if `*IN55` or `*ON` have no default value, because I am not so sure it is right that in this case the program must throw an error

Reply:
I'll ask!

Reply:
I run it on AS400 and this is the result:

![Schermata del 2024-05-28 08-07-31](https://github.com/smeup/jariko/assets/130480196/9c4db4bd-9362-49c2-8944-bce3709bd021)

Reply:
Ok